src/main/resources/assets/packagedauto/lang/ja_jp.json

```diff
@@ -0,0 +1,74 @@
+{
+	"itemGroup.packagedauto": "PackagedAuto",
+
+	"block.packagedauto.encoder": "パッケージレシピエンコーダー",
+	"block.packagedauto.encoder.save": "保存",
+	"block.packagedauto.encoder.save_single": "一つ保存",
+	"block.packagedauto.encoder.load": "読み込み",
+	"block.packagedauto.encoder.load_single": "一つ読み込み",
+	"block.packagedauto.encoder.pattern_slot": "レシピスロット %s",
+	"block.packagedauto.encoder.change_recipe_type": "レシピタイプ変更",
+	"block.packagedauto.packager": "パッケージャー",
+	"block.packagedauto.packager.mode.exact": "モード:正確性を要求",
+	"block.packagedauto.packager.mode.disjoint": "モード:不一致を要求",
+	"block.packagedauto.packager.mode.first": "モード:最初のマッチ",
+	"block.packagedauto.packager_extension": "パッケージャー拡張",
+	"block.packagedauto.unpackager": "アンパッケージャー",
+	"block.packagedauto.unpackager.blocking.false": "ブロッキングモードが無効",
+	"block.packagedauto.unpackager.blocking.true": "ブロッキングモードが有効",
+	"block.packagedauto.unpackager.tracker.decrease": "トラッカー数を減らす",
+	"block.packagedauto.unpackager.tracker.increase": "トラッカー数を増やす",
+	"block.packagedauto.distributor": "ポジションパッケージディストリビューター",
```

Comment:
The translation of "Positioned" here should probably be consistent with the recipe type